### PR TITLE
rofi: add plugins

### PR DESCRIPTION
**Pull request by luc65r**
I've added an option that rebuilds the rofi wrapper with specific plugins

This PR needs https://github.com/NixOS/nixpkgs/pull/83136 to be merged first for this to work

Example of this feature in use:

```nix
{ pkgs, ... }:

{
programs.rofi = {
  enable = true;
  plugins = with pkgs; [ rofi-calc ];
};
}
```

### CODE REVIEW

File: modules/programs/rofi.nix [hunk 1]
```diff
@@ -295,7 +303,7 @@ in {
       '';
     }];
 
-    home.packages = [ pkgs.rofi ];
+    home.packages = [ (pkgs.rofi.override({ inherit (cfg) plugins; })) ];
```
Comment (rycee) [inline modules/programs/rofi.nix]:
```suggestion
    home.packages = [ (pkgs.rofi.override { inherit (cfg) plugins; }) ];
```

**Comment by luc65r** [comment]
The PR on nixpkgs has been merged, so it should work now.

**Comment by rycee** [comment]
Looks good, thanks! To avoid breaking peoples' configuration I'd like for it to be available in the NixOS and Nixpkgs unstable channels for a week or so before merging.

**Comment by luc65r** [comment]
rofi-calc is in nixpkgs-unstable, but not in nixpkgs-20.03. Is that a problem?

**Comment by Sir4ur0n** [comment]
Hi, why was this work closed please? The idea and implementation seemed good :disappointed:

**Comment by luc65r** [comment]
#1225 solved the issue, even if this pr makes it a bit easier to use plugins.

**Comment by Sir4ur0n** [comment]
Ah right, thank you, I don't know how I missed that part of the appendix, sorry!

**Comment by luc65r** [comment]
I should have specified it when I closed this pr

**Pull request closed without merging**